examples/protocols/pppos_client/components/modem/src/bg96.c

```diff
@@ -210,6 +210,11 @@ static esp_err_t bg96_handle_cops(modem_dce_t *dce, const char *line)
                 err = ESP_OK;
             }
         }
+        if (i >= 4) {
+            char act_str[3] = {0};
+            int len = snprintf(act_str, 3, "%s", p[3]);
```

Comment:
nitpick: This will produce defined but not used warning. Do we want to check if the `len` is *correct*?

Reply:
Done